ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/network/DefaultP2PNetwork.java

```diff
@@ -228,7 +227,7 @@ public void start() {
                       disco,
                       createDaemonListener(),
                       0L,
-                      60000L,
+                      1800000L,
```

Comment:
If the call to the DNS server fails, this introduces a big time lag before we try again.

Reply:
good point. maybe somewhere in between?